tools/integration_tests/run_e2e_tests.sh

```diff
@@ -239,6 +239,25 @@ function run_e2e_tests_for_hns_bucket(){
    return 0
 }
 
+function run_e2e_tests_for_anywhere_cache(){
+   local test_bucket_name="golang-grpc-test-lankita-anywhere-cache-test" ## TODO: adding temporary bucket name for testing purpose
+
+   echo "Running tests for Anywhere Cache"
+   run_non_parallel_tests TEST_DIR_HNS_GROUP "$test_bucket_name" ## TODO: separate group will be created for anywhere cache in final tests
```

Comment:
nit: Should we add TODO above the code line?

Reply:
This will be updated in final PR